packages/cli/src/util/build/scrub-argv.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Redacts `--env`, `--build-env`, and `--token` in `argv`.
+ */
+export function scrubArgv(argv: string[]) {
```

Comment:
Let's re-use this function in `report-error.ts`?

Reply:
That code mutates `argv` post-parsing where the `build.json` contains `process.argv`. In other words, `report-error` code won't work without modifications at which point we're probably back to where we started.

Reply:
I think we could just do `args(scrubArgv(process.env))` and it would be the same thing.

Reply:
@TooTallNate does the build container do anything with `argv` from the `build.json` or is it just for debugging?

Reply:
Build container doesn't do anything with it.